pkgs/development/compilers/vlang/default.nix

```diff
@@ -76,11 +76,6 @@ stdenv.mkDerivation {
     cp -r ${boehmgcStatic}/lib/* ./thirdparty/tcc/lib
   '';
 
-  # vcreate_test.v requires git, so we must remove it when building the tools.
```

Comment:
Doesn't really need to be done in this PR, but we should see if we can prefetch whatever ``vcreate_test.v`` needs from whatever github repo instead of removing it if possible.